src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.netstandard2.0.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Reflection.Metadata
+{
+    public sealed partial class MetadataReader
+    {
+        internal AssemblyName GetAssemblyName(StringHandle nameHandle, Version version, StringHandle cultureHandle, BlobHandle publicKeyHandle, AssemblyHashAlgorithm assemblyHashAlgorithm, AssemblyFlags flags)
+        {
+            string name = (!nameHandle.IsNil) ? GetString(nameHandle) : null;
+            string cultureName = (!cultureHandle.IsNil) ? GetString(cultureHandle) : null;
+            var hashAlgorithm = (Configuration.Assemblies.AssemblyHashAlgorithm)assemblyHashAlgorithm;
+            byte[] publicKeyOrToken = !publicKeyHandle.IsNil ? GetBlobBytes(publicKeyHandle) : null;
+
+            var assemblyName = new AssemblyName(name)
+            {
+                Version = version,
+                CultureName = cultureName,
+                HashAlgorithm = hashAlgorithm
+            };
+            assemblyName.SetPublicKey(publicKeyOrToken);
+
+            assemblyName.Flags = GetFlagsFromAssemblyFlags(flags);
+            assemblyName.ContentType = GetContentTypeFromAssemblyFlags(flags);
+
+            return assemblyName;
+        }
+
+        private AssemblyNameFlags GetFlagsFromAssemblyFlags(AssemblyFlags flags)
+        {
+            AssemblyNameFlags assemblyNameFlags = AssemblyNameFlags.None;
```

Comment:
I don't think this is necessary as since this is a `[Flag]` enum by default the bits will be all set to 0, so it will already have the `.None` flag.